code/modules/projectiles/guns/ballistic.dm

```diff
@@ -262,11 +262,11 @@
 	if (bolt_type == BOLT_TYPE_OPEN)
 		if(!bolt_locked) //If it's an open bolt, racking again would do nothing
 			if (user)
-				to_chat(user, span_notice("[src]'s [bolt_wording] is already cocked!"))
+				balloon_alert(user, "[bolt_wording] already cocked")
```

Comment:
```suggestion
				balloon_alert(user, "[bolt_wording] already cocked!")
```

"!" indicates failure. Also, what is bolt_wording defined as in the usual places? I'm wondering if it fits balloon alert wording rules

Reply:
Add "!" everywhere you are indicating failure

Reply:
`bolt_wording` is usually like "bolt", "slide", etc. It should fit the wording rules from what I tested.

Also the wording guide says no trailing punctuation, does that not included exclamation marks?

Reply:
I need to update whatever you're reading.

Normal actions have no punctuation.
Failure actions end with `!`
Pending actions (such as before a do_after) end with `...`

Reply:
This is what I am reading: https://hackmd.io/@tgstation/SJIle5wYd#balloon_alertballoon_alert_to_viewers

If there's more current documentation please point me to it, I couldn't find anything.